src/pages/queue.js

```diff
@@ -63,9 +73,28 @@ class Queue extends React.Component {
       return <Error statusCode={404} />
     }
     const locationText = this.props.queue.location || 'No location specified'
+    const confidentialMessage = this.props.isUserCourseStaff
```

Comment:
We'll want a `|| isUserAdmin` here as well so that the message is correct for admins